src/components/ProductListing.vue

```diff
@@ -1,22 +1,79 @@
 <template>
 <section>
-    <nav class="">
-		<div class="fixed inset-0 overflow-hidden flex justify-between items-center px-36 h-20 bg-emerald-100  shadow-md mb-10 sm:px-10">
-            <div>
-                <p class="font-pacifico text-3xl">YourChoice</p>
+    <main>
+		<div class="mt-28 mx-36 sm:mx-10">
+            <div class="productlist grid grid-cols-4 gap-5 sm:grid-cols-2 " id="productlist">
+                <div v-for="(product, index) in paginatedItems" :key="index">
+                    <div class="border flex flex-col justify-center py-8 cursor-pointer">
+                        <div class="flex justify-center items-center">
+                            <img :src="product.thumbnail" class="h-64 w-80">
+                        </div>
+                        <div class="px-10">
+                            <p>{{product.title}}</p>
+                            <p class="font-bold text-emerald-500">Rs.{{calculateDiscoutedAmount(product)}}</p>
+                            <div class="flex">
+                                <p class="text-gray-400 mr-3 line-through text-sm">{{product.price}}</p>
+                                <p class="text-sm">-{{product.discountPercentage}}%</p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
```

Comment:
this could be extracted to a Product.vue component

Reply:
done